index.js

```diff
@@ -68,7 +76,15 @@ function checkIfAnyMovieHasRating() {}
       // Toy Story 4
     };
  */
-function findById() {}
+function findById(movies, id) {
+  if (movies.length === 0) { throw "No ID found" }
+  let movieId = movies.find(movies => movies.imdbID === id)
+  if (movieId === undefined)
+    return null
+  else return movieId
```

Comment:
You don't need an `else` here, since the `if` returns. If the condition is true, we'll never get to _this_ return, making them either/or already.